base/frontend/sourcegraph-frontend.Role.yaml

```diff
@@ -11,6 +11,7 @@ rules:
   - ""
   resources:
   - endpoints
+  - services
```

Comment:
can we include a comment here about why we need services? (and retro-acetively why we need endpoints?)

Reply:
done